client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -125,6 +126,9 @@ const siteMigration: Flow = {
 			window.location.assign( to );
 		};
 
+		if ( urlQueryParams.get( 'ref' ) === 'move-lp' ) {
+			recordMigrationStart();
```

Comment:
This looks good overall. The one thing I'm wondering about, related to performance, is whether this can be handled as a side effect so it doesn't block the step. I think Stepper has some utilities for this.

Reply:
I'll check it out